quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.upgrad.quora.api.controller;
```

Comment:
There are no comments in any of the files added/updated. Please add necessary comments

Reply:
Added